music.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  def initialize(on_spotify: true)
+    super()
```

Comment:
### [Optinal]:
Good job! you did a great job but try to make sure that this `super() `method is not reviving any parameter. I think you should send a `publish_date` to the super(). So please check and make sure.